index.bs

```diff
@@ -2370,10 +2389,14 @@ defined as follows:
               and |device| does not <a>match a device filter</a> in
               <code>|desc|.{{USBPermissionDescriptor/filters}}</code>, continue
               to the next |device|.
-           2. Get the {{USBDevice}} representing |device| and add it to
+           2. If <code>|desc|.{{USBPermissionDescriptor/exclusionFilters}}</code>
+              is set and |device| does <a>match a device filter</a> in
```

Comment:
```suggestion
              is set and |device| <a>matches a device filter</a> in
```

You might need to add an attribute to the definition of "match a device filter" for it to understand this version is the same definition. I forget how smart it is about this.